src/utils/formatting-helpers.js

```diff
@@ -13,8 +13,12 @@ export const momentUTCFormat = (d, f) =>
     .utc(d)
     .locale("en")
     .format(f)
-export const genericDateTimeFormat = d =>
-  `${momentUTCFormat(d, "MMM D, YYYY")} \u205F${momentUTCFormat(d, "HH:mm:ss")}`
+export const genericDateTimeFormat = d => {
+  if (d.getMilliseconds() === 0) {
```

Comment:
For now, we're truncating millisecond timestamps is the `d.getMilliseconds() === 0`